### PR TITLE
Add support for Swift Package Manager

### PR DESCRIPTION
**Pull request by RafaelFernandezAlv**
Hi.

I developed the support for use this library with Swift Package Manager. Can you merge and generate the tag 3.1.0?

Thanks!

### CODE REVIEW

**Comment by mathaeus** [comment]
Hi, thanks for adding SPM support, is there any update on this? @RafaelFernandezAlv @Truba

**Comment by rromanchuk** [comment]
@RafaelFernandezAlv using your fork now. No problems, thanks.

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            targets: ["JapxRxCodableAlamofire"]),
+        .library(
+            name: "JapxRxCodableMoya",
+            targets: ["JapxRxCodableMoya"]),
+        .library(
+            name: "JapxRxMoya",
+            targets: ["JapxRxMoya"]),
+        .library(
+            name: "JapxObjC",
+            targets: ["JapxObjC"])
```
Comment (rromanchuk) [inline Package.swift:44]: duplicate with https://github.com/infinum/Japx/pull/42/files#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eR27

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    ],
```
Comment (rromanchuk) [inline Package.swift:45]:
```suggestion
            targets: ["JapxRxMoya"])
    ],
```

**Review by rromanchuk — CHANGES_REQUESTED** [review]

**Pull request merged**